charlotte_core/src/main.rs

```diff
@@ -5,16 +5,18 @@
 mod arch;
 mod bootinfo;
 mod framebuffer;
+mod memory;
 
 use core::fmt::Write;
 
-use arch::x86_64::memory::pmm::PFA;
 use arch::{Api, ArchApi};
 
 use framebuffer::console::CONSOLE;
 
 use crate::framebuffer::framebuffer::FRAMEBUFFER;
 
+use memory::pmm::*;
```

Comment:
Nit: Should probably be avoiding wilcard imports but fine for testing purposes.

Reply:
I can change it to only pull in what's used.